### DISCUSSION TITLE
How to use external Redis and PostgreSQL

### DISCUSSION THREAD

**Discussion by nikitazep** [mediacms-io/mediacms#965] [Q&A]
I'm using unraid system and have Redis and PostgreSQL installed.
If you install mediacms you need to install Redis and PostgreSQL again.
So how to use existing Redis and PostgreSQL?

**Comment by nikitazep** [comment]
How to set it up in docker compose

**Comment by hellopanictech** [comment]
I'm using unRAID as well and would love to use this software. I've tried making a template, but can't get things to line up. Being able to use our own external Redis and PostgreSQL would surely help on that path.
